### DISCUSSION TITLE
have a option to hide the pp counter in the results screen

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#13148] [Ideas]
currently, the pp counter is displayed on the results screen(but not working)

the option to hide said counter would be for people who don't want to get pressured by being rated for their performance and just want to have fun

you can unhide it if you want to see your performances rated

your performances will still be rated but you don't have to see how much of a rating you got

**Comment by bdach** [comment]
Not sure I see the point. If you find the counter distracting then don't look at it. Especially now that it's solely for display (but it should be working, contrary to that the above post claims).

**Comment by Game4all** [comment]
The non-working part is being tracked at  #8587

**Comment by iSimpFor** [comment]
i feel like it would reduce pressure to those who want to stop caring about pp but also can't stop caring
i think simply having a counter there makes people more likely to care about it

**Comment by peppy** [comment]
I don't believe this will be a thing. If anything, it could be considered for when the relax mod is active.

**Comment by iSimpFor** [comment]
if you do hide it for relax, hide it for autopilot too

**Comment by Game4all** [comment]
i think the point was to disable the pp counter when a unranked mod is enabled
